ginivision/src/main/java/net/gini/android/vision/digitalinvoice/details/LineItemDetailsActivity.kt

```diff
@@ -6,28 +6,70 @@ import android.os.Bundle
 import android.view.MenuItem
 import androidx.appcompat.app.AppCompatActivity
 import net.gini.android.vision.R
+import net.gini.android.vision.camera.CameraActivity
+import net.gini.android.vision.digitalinvoice.DigitalInvoiceActivity
 import net.gini.android.vision.digitalinvoice.SelectableLineItem
 import net.gini.android.vision.internal.util.ActivityHelper.enableHomeAsUp
 
 private const val LINE_ITEM_DETAILS_FRAGMENT = "LINE_ITEM_DETAILS_FRAGMENT"
 private const val EXTRA_IN_SELECTABLE_LINE_ITEM = "EXTRA_IN_SELECTABLE_LINE_ITEM"
 
+/**
+ * When you use the Screen API, the `LineItemDetailsActivity` displays a line item to be edited by the user. The user can modify the
+ * following:
+ * - deselect the line item,
+ * - edit the line item description,
+ * - edit the quantity,
+ * - edit the price.
+ *
+ * The extractions returned in the [CameraActivity.EXTRA_OUT_COMPOUND_EXTRACTIONS] are updated to include the user's modifications.
+ *
+ * The `LineItemDetailsActivity` is started by the [DigitalInvoiceActivity] when a user taps on a
+ * line item to edit it.
+ *
+ * ### Customizing the Line Item Details Screen
+ *
+ * Customizing the look of the Line Item Details Screen is done via overriding of app resources.
+ *
+ * The following items are customizable:
+ * - TODO
+ * - TODO
```

Comment:
TODO here as well

Reply:
Customization stuff I want to do in this ticket https://ginis.atlassian.net/browse/PIA-595